src/applications/disability-benefits/2346/containers/App.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+import RoutedSavableApp from '../../../../platform/forms/save-in-progress/RoutedSavableApp';
+import formConfig from '../config/form';
+
+export default function App({ location, children }) {
+  return (
+    <RoutedSavableApp formConfig={formConfig} currentLocation={location}>
```

Comment:
I'm not sure that firing up a multipage benefits application app with `RoutedSavableApp` and a `formConfig` is the best way to approach this application. Perhaps a better model for the structure of your app would be something like the Dashboard app in `src/applications/personalization/dashboard`